src/js/modules/utilities/domain.ts

```diff
@@ -0,0 +1,9 @@
+import { Ntp } from "../domain/generatedRpc/generatedClasses";
+
+export const isNtpEqual = (ntp1: Ntp, ntp2: Ntp): boolean => {
```

Comment:
Can we just overload operator == for the custom types?

Reply:
NodeJs doesn't allow override operators